scout/server/blueprints/clinvar/controllers.py

```diff
@@ -276,7 +276,7 @@ def parse_variant_form_fields(form):
         clinvar_var["variations_ids"] = form["dbsnp_id"]
 
     if clinvar_var.get("ref_seq") and clinvar_var.get("hgvs"):
-        # Variant is described by RefSeq and HGVS already, remove redundanti fields from submission
+        # Variant is described by RefSeq and HGVS already, remove redundant fields from submission
```

Comment:
😄

Reply:
No, the dry run doesn't check that type of things it seems (I tried now).

Checked on their docs and they say: 
<img width="994" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/13a4448f-1720-4937-9efa-a02505eae410">

So we should be fine, no?